rs/canister_sandbox/src/process.rs

```diff
@@ -102,32 +102,35 @@ pub fn spawn_canister_sandbox_process_with_factory(
 
     // Set up thread to handle incoming channel -- replies are routed
     // to reply buffer, requests to the RPC request handler given.
-    let thread_handle = std::thread::spawn(move || {
-        let demux = transport::Demux::<_, _, protocol::transport::SandboxToController>::new(
-            Arc::new(rpc::ServerStub::new(
-                controller_service,
-                out.make_sink::<protocol::ctlsvc::Reply>(),
-            )),
-            reply_handler.clone(),
-        );
-        transport::socket_read_messages::<_, _>(
-            move |message| {
-                demux.handle(message);
-            },
-            socket,
-            SocketReaderConfig::default(),
-        );
-        reply_handler.flush_with_errors();
-        // If we the connection drops, but it is not terminated from
-        // our end, that implies that the sandbox process died. At
-        // that point we need to terminate replica as we have no way
-        // to progress execution safely, and we can not restart
-        // execution in a deterministic and safe manner that will not
-        // corrupt the state.
-        if !safe_shutdown.load(Ordering::SeqCst) {
-            abort_and_shutdown();
-        }
-    });
+    let thread_handle = std::thread::Builder::new()
+        .name("CanisterSandbox".to_string())
+        .spawn(move || {
```

Comment:
Could we call this one `CanisterSandboxIPC` just to clarify that it's code running in the replica for sandbox communication and not the sandbox itself?